### PR TITLE
FIX: makes sure an hex is always at 6 chars

### PR DESCRIPTION
**Pull request by jjaffeux**
eg: #aaa should be converted to #aaaaaa

### CODE REVIEW

File: app/models/report.rb [hunk 1]
```diff
@@ -1325,6 +1325,15 @@ def hex_to_rgbs(hex_color)
   end
 
   def rgba_color(hex, opacity = 1)
+    if hex.length === 3
```
Comment (ZogStriP) [inline app/models/report.rb]: Why not do?

```ruby
hex += hex if hex.size == 3
```

or 

```ruby
hex *= 2 if hex.size == 3
```

Comment (ZogStriP) [reply app/models/report.rb]: Also, `===` does not mean what you think it means in Ruby ;) I guess too much JavaScript :P

Comment (jjaffeux) [reply app/models/report.rb]: For #f49 for example the expected result is : `#ff4499`, you double each letter

```
hex = "f49"
chars = hex.scan(/\w/)
p chars.zip(chars).flatten.join.ljust(6) #  "ff4499"

hex = "f49"
p hex += hex # "f49f49"

hex = "f49"
p hex *= 2 # "f49f49"
```

Comment (ZogStriP) [reply app/models/report.rb]: Right. Was thinking about the easy case "000". Cary on!

**Pull request merged**